samples/zoomable/src/main/java/com/facebook/samples/zoomable/ZoomableDraweeView.java

```diff
@@ -115,6 +116,33 @@ private void init() {
     mZoomableController = createZoomableController();
     mZoomableController.setListener(mZoomableListener);
     mTapGestureDetector = new GestureDetector(getContext(), mTapListenerWrapper);
+    mTapGestureDetector.setOnDoubleTapListener(new GestureDetector.OnDoubleTapListener() {
```

Comment:
It would be better to make this settable via ZoomableDraweeView instead and have a separate class for the double tap listener.